src/encoder.rs

```diff
@@ -1106,6 +1106,22 @@ pub fn motion_compensate<T: Pixel>(
   }
 }
 
+pub fn save_block_motion<T: Pixel>(
+   fi: &FrameInvariants<T>, fs: &mut FrameState<T>,
```

Comment:
Maybe take in directly `w_in_b` and `h_in_b`, as you did for `get_subset_predictors`?